### ISSUE TITLE
Multiple choice 

### ISSUE THREAD

**Issue by mareekuh** [microsoft/BotFramework-Composer#6168]
in a flow, Add a question > multiple choice > edit properties

In the 'user input' tab, choose 'List' in the dropdown for list style 
Notice that dropdown arrow disappears - you can still click to get the dropdown but it doesn't show

![Screen Shot 2021-03-03 at 4 32 29 PM](https://user-images.githubusercontent.com/48503434/109891594-31305600-7c3e-11eb-98bc-9a44d01e7390.png)

**Comment by tdurnford** [comment]
@sangwoohaan Can you comment?

**Comment by sangwoohaan** [comment]
This should be always a dropdown no matter what you choose by design. Must be a bug

**Comment by tdurnford** [comment]
@sangwoohaan I don't believe there is a way to make it a text field and a dropdown at the same time.

**Comment by sangwoohaan** [comment]
It's not a text field, isn't it? You can only choose from one of the list types.

**Comment by mareekuh** [comment]
it's just that the dropdown disappears but it should still be a dropdown not a text entry box.

**Comment by tdurnford** [comment]
For List Styles, you can either select from a list of items - "none", "auto", "inline", "list", "suggestedAction", or "heroCard" - or enter an expression. For the default list, the input field needs to be a dropdown. For the expression, the input field needs to be an expression. 

Here is the schema for list style.
```
"style": {
  "$role": "expression",
  "title": "List style",
  "description": "Sets the ListStyle to control how choices are rendered.",
  "oneOf": [
    {
      "type": "string",
      "title": "Standard style",
      "description": "Standard style for rendering choices.",
      "enum": [
        "none",
        "auto",
        "inline",
        "list",
        "suggestedAction",
        "heroCard"
      ],
      "default": "auto"
    },
    {
      "$ref": "#/definitions/equalsExpression"
    }
  ]
},
```

**Comment by tdurnford** [comment]
Correct. That is how the component works currently.

**Comment by tdurnford** [comment]
Correct. That is how the component works currently.

**Comment by sangwoohaan** [comment]
It turns into a text field when I selected any option other than Write an expression right now. If you look at the condition field in If condition action node, that's how this list is supposed to work(even though they are different data types)

**Comment by tdurnford** [comment]
Oh, strange. I'll take a look at that. You're right, if the value is included in the options, it should remain a dropdown.
